defaults/main.yml

```diff
@@ -29,6 +29,7 @@ react_env_name: "prod"
 react_app_name: "{{ react_system_user }}"
 react_codebase_path: "{{ react_system_user_home }}/app"
 react_versioned_path: "{{ react_codebase_path }}-versioned"
+react_transpile_packages: true
```

Comment:
Let's make this false by default.  Once we do so we will also need to add a test scenario that has set it to true.